src/framework/parsers/texture/img-alpha-test.js

```diff
@@ -0,0 +1,86 @@
+import { PIXELFORMAT_RGBA8 } from '../../../platform/graphics/constants.js';
+import { RenderTarget } from '../../../platform/graphics/render-target.js';
+import { Texture } from '../../../platform/graphics/texture.js';
+
+//
+// Current state of ImageBitmap (April 2023):
+//
+// Chrome MacOS and Android (Pixel 3a and Pixel 7 Pro):
+//   Correctly loads PNG alpha and runs faster (significantly so on mobile) than HTMLImageElement.
+//
+// Firefox MacOS:
+//   Correctly loads PNG alpha, but runs significantly slower than HTMLImageElement.
```

Comment:
You mentioned a re-test of this showed the performance difference was perhaps not as significant as you thought. Is this still 'accurate'?

Reply:
Firefox timings are a little noisy, but I'm seeing 300ms vs 200ms load time on a test scene on MacOS.